package.json

```diff
@@ -161,6 +161,8 @@
     "parse-github-url": "1.0.3",
     "parse-gitignore": "2.0.0",
     "path-key": "4.0.0",
+    "pnpm": "^9.12.2",
+    "prettier": "^3.3.3",
```

Comment:
why were these added for this PR?

Reply:
I believe we were working on an new machine, and they were accidently not install locally, they have been removed in the update.